src/Adapter/Form/ChoiceProvider/TaxRuleGroupChoiceProvider.php

```diff
@@ -53,19 +56,33 @@ public function getChoices()
      */
     public function getChoicesAttributes(): array
     {
-        $attrs = [];
-        foreach ($this->getRules() as $rule) {
-            // Keep first one found
-            if (!empty($attrs[$rule['name']]['data-tax-rate'])) {
-                continue;
-            }
+        $address = new Address();
+        $address->id_country = (int) Context::getContext()->country->id;
```

Comment:
We try to avoid using `Context::getContext` directly in recent code, This should be injected in the provider instead, you can inject the `LegacyContext` in this service, or (maybe better) simply inject the country ID this way the service is not bound to any context service but only the data that is relevant for its use (which will simplify future refacto, since only the service definition ill need to be updated).


In the service you can inject something like `- "@=service("prestashop.adapter.legacy.context").getContext().country.id"`